### PR TITLE
Solving SonarQube issues (part two)

### PR DESCRIPTION
**Pull request by MithraicMagic**

### CODE REVIEW

File: osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/utils/FindMatchingChannelHelper.java [hunk 1]
```diff
@@ -99,7 +99,7 @@ private static boolean validateMbusDeviceInfoMatch(final MbusChannelElementsDto
     private static boolean hasValidMbusAttributes(final MbusChannelElementsDto mbusChannelElements,
             final ChannelElementValuesDto channelElementValues) {
         return channelElementValues.isMbusSlaveDeviceConfigured()
-                && mbusChannelElements.getPrimaryAddress().equals(mbusChannelElements.getPrimaryAddress());
+                && mbusChannelElements.getPrimaryAddress().equals(channelElementValues.getPrimaryAddress());
```
Comment (robindenadel) [inline osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/utils/FindMatchingChannelHelper.java:102]: why did you change this?

Comment (MithraicMagic) [reply osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/utils/FindMatchingChannelHelper.java:102]: The equals method is here used to compare mbusChannelElements.getPrimaryAddress() to mbusChannelElements.getPrimaryAddress(), which is exactly the same object (comparing is redundant).

Looking at the other methods in this class, I had assumed that this was a copy/paste mistake and that the mbusChannelElements were supposed to be compared to the channelElementValues.

**Review by robindenadel — APPROVED** [review]
